label.rb

```diff
@@ -0,0 +1,18 @@
+require_relative './item'
+
+class Label
+  attr_reader :id, :title, :color, :items
+
+  def initialize(id, title, color)
+    super(id)
```

Comment:
Kindly, check this line again. We use super to call constructor of parent class. As shown on the class diagram, this is not a  drived class. Hence, you may remove this line